docs/examples/FDR.ipynb

```diff
@@ -0,0 +1,1263 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ef4bdd06",
+   "metadata": {},
+   "source": [
+    "# Run CoCiP over FDR or QAR data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "123564d5",
+   "metadata": {},
+   "source": [
+    "## References\n",
+    "- Schumann, U. “A Contrail Cirrus Prediction Model.” Geoscientific Model Development 5, no. 3 (May 3, 2012): 543–80. https://doi.org/10.5194/gmd-5-543-2012.\n",
+    "- EASA: ICAO Aircraft Engine Emissions Databank (07/2021), 2021. \n",
+    "- ICAO: Annex 16: Environmental Protection - Volume II - Aircraft Engine Emissions: https://store.icao.int/en/annex-16-environmental-protection-volume-ii-aircraft-engine-emissions, 2008. Last access: 18 August 2022. "
```

Comment:
@trdean1 this link was broken. Can you confirm I found the right link?


Note I also have this file as a reference: https://www.icao.int/environmental-protection/CarbonOffset/Documents/Methodology%20ICAO%20Carbon%20Calculator_v11-2018.pdf

Reply:
Yes, that looks right. Let me find where I got this from because that link will be broken too...